cdap-ui/app/hydrator/controllers/create/partials/nodeconfig-ctrl.js

```diff
@@ -446,6 +446,85 @@ class HydratorPlusPlusNodeConfigCtrl {
     }
     return true;
   }
+  validatePluginPreferences() {
```

Comment:
nit: let's call this `validatePluginProperties`. Preferences is a different thing in CDAP